strawberry/subscriptions/__init__.py

```diff
@@ -1,2 +1,9 @@
+from typing_extensions import Literal
+
 GRAPHQL_TRANSPORT_WS_PROTOCOL = "graphql-transport-ws"
 GRAPHQL_WS_PROTOCOL = "graphql-ws"
+
+SubscriptionProtocolType = Literal["graphql-transport-ws", "graphql-ws"]
+
+# Code 4406 is "Subprotocol not acceptable"
+WS_4406_PROTOCOL_NOT_ACCEPTABLE = 4406
```

Comment:
I like this constant, but I think it should live in "strawberry.subscriptions.protocols.graphql_transport_ws" since it's specific to that protocol. What do you think?